docs/actions.md

```diff
@@ -120,29 +120,65 @@ Parameters can be passed to the action when it is invoked.
   ```
   $ wsk action update hello hello.js
   ```
+
+3.  Parameters can be provided explicitly on the command-line, or by supplying a file containing the desired parameters
+
+  To pass parameters directly through the command-line, supply a key/value pair to the ```--param``` flag:
   ```
   $ wsk action invoke --blocking --result hello --param name 'Bernie' --param place 'Vermont'
   ```
+
+  In order to use a file containing parameter content, create a file containing the parameters in JSON format. The
+  filename must then be passed to the ```param-file``` flag:
+
+  Example parameter file called actionParams.json:
+  ```
```

Comment:
Using triple ticks is OK for multiline, but not for a word in a sentence  line ``param-file``